modules/govuk/manifests/apps/content_data_api/rabbitmq.pp

```diff
@@ -55,6 +57,69 @@
     durable       => true,
   }
 
+  govuk_rabbitmq::queue_with_binding { $amqp_queue_temp:
+    ensure        => 'present',
+    amqp_exchange => $amqp_exchange,
+    amqp_queue    => $amqp_queue_temp,
+    routing_key   => '*.major',
+    durable       => true,
+    arguments     => {
+      x-dead-letter-exchange => $amqp_dlx,
+    },
+  }
+
+  rabbitmq_binding { "binding_minor_${amqp_exchange}@${amqp_queue_temp}@/":
+    ensure           => 'present',
+    name             => "${amqp_exchange}@${amqp_queue_temp}@minor@/",
+    user             => 'root',
+    password         => $::govuk_rabbitmq::root_password,
+    destination_type => 'queue',
+    routing_key      => '*.minor',
+    arguments        => {},
+  }
+
+  rabbitmq_binding { "binding_links_${amqp_exchange}@${amqp_queue_temp}@/":
+    ensure           => 'present',
+    name             => "${amqp_exchange}@${amqp_queue_temp}@links@/",
+    user             => 'root',
+    password         => $::govuk_rabbitmq::root_password,
+    destination_type => 'queue',
+    routing_key      => '*.links',
+    arguments        => {},
+  }
+
+  rabbitmq_binding { "binding_republish_${amqp_exchange}@${amqp_queue_temp}@/":
+    ensure           => 'present',
+    name             => "${amqp_exchange}@${amqp_queue_temp}@republish@/",
+    user             => 'root',
+    password         => $::govuk_rabbitmq::root_password,
+    destination_type => 'queue',
+    routing_key      => '*.republish',
+    arguments        => {},
+  }
+
+  rabbitmq_binding { "binding_unpublish_${amqp_exchange}@${amqp_queue_temp}@/":
+    ensure           => 'present',
+    name             => "${amqp_exchange}@${amqp_queue_temp}@unpublish@/",
+    user             => 'root',
+    password         => $::govuk_rabbitmq::root_password,
+    destination_type => 'queue',
+    routing_key      => '*.unpublish',
+    arguments        => {},
+  }
+
+  govuk_rabbitmq::queue_with_binding { $amqp_bulk_importing_queue_temp:
+    ensure        => 'present',
+    amqp_exchange => $amqp_exchange,
+    amqp_queue    => $amqp_bulk_importing_queue_temp,
+    routing_key   => '*.bulk.data-warehouse',
+    durable       => true,
+    arguments     => {
+      x-dead-letter-exchange => "${amqp_dlx}@/",
+    },
+  }
```

Comment:
This is probably unnecessary right, as we can just avoid enqueuing jobs while switching the queues around?